lib/modules/datasource/go/base.ts

```diff
@@ -207,7 +208,7 @@ export class BaseGoDatasource {
       .join('/');
 
     return {
-      datasource: GithubTagsDatasource.id,
+      datasource: GitTagsDatasource.id,
       registryUrl: `${parsedUrl.protocol}//${parsedUrl.host}`,
```

Comment:
git-tags does not support registryUrls, so I don't think the change is as simple as this. However I thought this had been tested?

Reply:
The field `registryUrls` is required in order to comply with the `DataSource` interface. You are right that I was missing some things though.